src/ILVerify/src/VerifierError.cs

```diff
@@ -85,7 +85,7 @@ enum VerifierError
         //E_THIS_UNINIT_V_RET  "Return from .ctor before all fields are initialized."
         //E_THIS_UNINIT_BR     "Branch back when this is uninitialized."
         //E_LDFTN_CTOR         "ldftn/ldvirtftn not allowed on .ctor."
-        //E_STACK_NOT_EQ       "Non-compatible types on the stack."
+        E_STACK_NOT_EQ,                 // "Non-compatible types on the stack."
```

Comment:
Could you please make the names of the error messages CamelCase as you enable them?